.circleci/cimodel/data/pytorch_build_definitions.py

```diff
@@ -136,6 +136,20 @@ def gen_workflow_job(self, phase):
     def gen_build_name(self, _):
         return self.name
 
+class DocPushConf(object):
+    def __init__(self, name, parent_build=None):
+        self.name = name
+        self.parent_build = parent_build
+
+    def gen_workflow_job(self, phase):
+        return {
+            "pytorch_doc_push": {
+                "name": self.name,
+                "requires": [self.parent_build],
+                "context": "org-member",
+                "filters": gen_filter_dict(branches_list=["nightly"])
```

Comment:
This is up for debate on how often it should run, but I'm not entirely sure if circle supports the `org-member` context on the master branch